nifi-nar-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/src/main/java/org/apache/nifi/cdc/mysql/processors/CaptureChangeMySQL.java

```diff
@@ -368,6 +393,23 @@
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .build();
 
+    public static final PropertyDescriptor SSL_CONTEXT_SERVICE = new PropertyDescriptor.Builder()
+            .name("SSL Context Service")
+            .displayName("SSL Context Service")
+            .description("SSL Context Service supporting encrypted socket communication")
+            .required(false)
+            .identifiesControllerService(SSLContextService.class)
+            .build();
+
+    public static final PropertyDescriptor SSL_MODE = new PropertyDescriptor.Builder()
```

Comment:
Do we want to use the new .dependsOn functionality to display this property only if the SSL Context Service is set?

Reply:
@pvillard31 Thanks for following up with the recommendation.  I updated the `SSL Context Service` property descriptor to depend on the relevant values from the `SSL Mode` property.